src/main.rs

```diff
@@ -341,6 +377,8 @@ fn main() -> Result<(), Error> {
         println!("{}", op);
         if opts.metadata {
             op.print_metadata(&resolver, opts.changelog)?;
+        } else if opts.changelog {
```

Comment:
:thinking: Why should these two flags be exclusive? I think we don't want the `else` here, do we?

Reply:
This is here, so that we don't print the CHANGELOG twice.